tests/ats/test_helm_works.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from typing import List
+
+import pykube
+import pytest
+
+from pytest_helm_charts.k8s.fixtures import NamespaceFactoryFunc
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.flux.helm_release import (
+    HelmReleaseFactoryFunc,
+    ChartTemplate,
+    CrossNamespaceObjectReference,
+)
+from pytest_helm_charts.flux.helm_repository import HelmRepositoryFactoryFunc
+
+from helpers import assert_hello_world_is_running
+
+logger = logging.getLogger(__name__)
+
+APP_DEPLOYMENT_TIMEOUT_SEC = 180
+
+
+@pytest.mark.functional
+@pytest.mark.upgrade
+def test_helm_works(
```

Comment:
```suggestion
def test_helm_release_works(
```

Could you rename the file too? The current name is a bit ambiguous.